portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -232,8 +247,13 @@
 	<td></td>
 </tr>
 <tr>
-	<td>HEADER_ROW</td>
-	<td>xpath=(//div[contains(@class, "dnd-tr")])[1]</td>
+	<td>DELETE_VIEW_BUTTON</td>
+	<td>//button[@class='dropdown-item'][normalize-space(text())='Delete View']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DELETE_BUTTON</td>
+	<td>//button[@class='btn btn-primary']</td>
```

Comment:
That path is not being used in the test 😄

Reply:
Hey @mateusralv, this path is being used on `deleteView` macro .